packages/atomicHelpers/persistentStorage.ts

```diff
@@ -0,0 +1,97 @@
+import type { PersistentAsyncStorage } from "@dcl/kernel-interface"
+
+declare var window: any
+
+class PersistentLocalStorage implements PersistentAsyncStorage {
+  storage: any
+  constructor(storage: any){
+    if (storage) {
+      this.storage = storage
+    }else{
+      throw new Error("Cannot create PersistentLocalStorage without localStorage object")
+    }
+  }
+
+  clear(): Promise<void> {
+    console.log("PersistentLocalStorage::clear")
+    return new Promise(resolve => {
+      this.storage.clear()
+      resolve()
+    })
+  }
```

Comment:
For all async functions, there is no need to manually create `new Promise` if you add the `async` modifier to the function decl
```suggestion
  async clear(): Promise<void> {
    console.log("PersistentLocalStorage::clear")
    this.storage.clear()
  }
```